app/assets/stylesheets/components/vacancy_review_heading.scss

```diff
@@ -0,0 +1,14 @@
+@import 'base';
+
+.vacancy-review-heading-component {
```

Comment:
this should follow the naming of the 'root' component i.e `review-component__heading` and then can use ampersand notation `&__heading` if you relocated the import mentioned above